### PR TITLE
Initial Vue setup

### LINKED ISSUE #9: Vue setup
Initiales Vue Setup in einem `client/` Verzeichnis mit initialer Struktur:

- Vue 3 Installation
- Ordnerstruktur mit zumindest folgendem Aufbau in `src/`:
  - atoms
  - molecules
  - organisms
  - templates
  - pages
  - router
  - store
- Alle nicht benötigten initialen Vue Dateien (favicon, etc.) werden entfernt
- ggf. erste Komponenten in Verwendung

### PR DESCRIPTION
**Pull request by AntonijeStefanovic**
Closes #9

### CODE REVIEW

**Comment by AntonijeStefanovic** [comment]
Closes #9

File: client/src/atoms/BaseButton.vue [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<template>
+  <button class="btn">{{ text }}</button>
```
Comment (lukasfoidl) [inline client/src/atoms/BaseButton.vue]: Please already use the ionic library for all UI components (see https://ionicframework.com/docs/api/button).

File: client/src/atoms/BaseButton.vue [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+<style scoped>
+.btn {
+  background-color: #007bff;
+  color: white;
+  padding: 10px;
+  border: none;
+  border-radius: 4px;
+  cursor: pointer;
+}
```
Comment (lukasfoidl) [inline client/src/atoms/BaseButton.vue]: This style shouldn't be necessary with an ionic button.

**Review by lukasfoidl — CHANGES_REQUESTED** [review]
Very good work just small fixes mentioned in code and please add the remaining folders mentioned in the ticket to force others to use the given structure.

If you are interested you could already add some demonstration with routing and store usages so we can build upon this implementation how to use the router and the store later, but as it is not mentioned in the ticket, I will also merge it without it.

**Comment by lukasfoidl** [comment]
As #8 is now merged into main please combine your `README.md` with the `README.md` in the projects root folder (please keep consistent styling).

File: client/src/atoms/BaseButton.vue (hunk 1 above)
Comment (AntonijeStefanovic) [reply client/src/atoms/BaseButton.vue]: I have updated the `BaseButton.vue` component to use the Ionic button.

File: client/src/atoms/BaseButton.vue (hunk 2 above)
Comment (AntonijeStefanovic) [reply client/src/atoms/BaseButton.vue]: I have removed the custom css.

File: client/src/atoms/BaseButton.vue (hunk 1 above)
Comment (AntonijeStefanovic) [reply client/src/atoms/BaseButton.vue]: Added "About" and "Impressum" pages to demonstrate the routing.

**Review by lukasfoidl — APPROVED** [review]

**Pull request merged**